src/components/Annotation/Annotation.scss

```diff
@@ -0,0 +1,126 @@
+@import '../../common.scss';
+
+.ba {
+    .ba-annotation-container {
+        @import '~box-react-ui/lib/styles/common/inline-notifications';
+
+        border-bottom: 1px solid $seesee;
+        position: relative;
+        z-index: 1;
+
+        &:last-child {
+            border-bottom: 0;
+        }
+
+        .ba-annotation-error {
```

Comment:
Each time styles are nested, they increase specificity, which makes further extension and maintenance more difficult. Is this necessary here or can they be at the top level, nested directly under `.ba`?